src/middleware/logger.ts

```diff
@@ -1,6 +1,7 @@
 import chalk from 'chalk';
+import { NextFunction, Request } from 'express';
 
-export default (req, _, next) => {
+export default (req: Request, _: any, next: NextFunction) => {
```

Comment:
we don't want to use `any` - rather if we need to specify types (which I don't think we need to?) we should provide `Response`